apps/site/lib/site_web/templates/event/_calendar.html.eex

```diff
@@ -38,6 +38,17 @@
                   conn: @conn,
                   event: event,
                   class: if index >= 4, do: "open-left" %>
+
+                  <noscript>
+                    <%= link to: cms_static_page_path(@conn, event.path), class: "m-event-calendar__event m-event-calendar__event-title", title: event.title, target: "_blank" do %>
+                      <i class="fa fa-circle <%= if event_ended, do: 'ended' %>" aria-hidden="true"></i>
+                      <%= if !event_ended do%>
+                        <div class='m-event-calendar__event-time'><%= format_time(event.date) %></div>
+                      <% end %>
+                      <%= event.title %>
```

Comment:
Was it intentional to not use `<div class="m-event-calendar__event-title"><%= event.title %></div>` like in the regular display?

Reply:
Yes, because `m-event-calendar__event-title` is now being applied to the (parent) `<%= link ...` and so that div was no longer necessary.

Reply:
Ah, makes sense!